file_aloc/bl_import_aloc.py

```
@@ -0,0 +1,292 @@
+        # PhysicsCollisionLayerType.DYNAMIC_COLLIDABLES_ONLY_NO_CHARACTER_TRANSPARENT,
+        # PhysicsCollisionLayerType.DYNAMIC_TRASH_COLLIDABLES,
+        # PhysicsCollisionLayerType.HERO_DYN_BODY,
+        # PhysicsCollisionLayerType.ITEMS,
+        # PhysicsCollisionLayerType.KINEMATIC_COLLIDABLES_ONLY,
+        # PhysicsCollisionLayerType.KINEMATIC_COLLIDABLES_ONLY_TRANSPARENT,
+        # PhysicsCollisionLayerType.WEAPONS
+    ]
+    return collision_layer not in excluded_collision_layer_types
+
+
+def load_aloc(operator, context, filepath, include_non_collidable_layers):
```

Ruff (0.14.8)

Unused function argument: `operator`

(ARG001)